db/contacts.json

```diff
@@ -0,0 +1,134 @@
+{
+  "contacts": [
+    {
```

Comment:
@hagansmith - this should be an object of objects with keys for each contact. Annnd, while you are in here, you may as well add uid to contacts, right? 💁